test/logoutrequest_test.rb

```diff
@@ -91,18 +81,24 @@ class RequestTest < Minitest::Test
     end
 
     describe "when the settings indicate to sign (embedded) logout request" do
-      it "created a signed logout request" do
-        settings = OneLogin::RubySaml::Settings.new
-        settings.idp_slo_target_url = "http://example.com?field=value"
-        settings.name_identifier_value = "f00f00"
+
+      let(:settings2) { OneLogin::RubySaml::Settings.new }
```

Comment:
In the test I use 3 different settings: settings, settings2 and settings3, each initiated with different values in the before block ...

In this specific case I think that naming each value with a descriptive name like settings_for_test_signatures, settings_for_test_authcontext .. will do the code less legible due the code length, but I'm open to suggestions.